script.js

```diff
@@ -0,0 +1,241 @@
+console.log("Assignment TicTacToe")
+
+//initial prompt to get user input for their names
+var playerNameOne = prompt("Hi there player 1, what is your name?");
+var playerNameTwo = prompt("Hi there player 2, what is your name?");
+
+//retrieve element via their id and assign that element to a variable for use
+var one = document.querySelector("#zero-zero");
+var two = document.querySelector("#zero-one");
+var three = document.querySelector("#zero-two");
+var four = document.querySelector("#one-zero");
+var five = document.querySelector("#one-one");
+var six = document.querySelector("#one-two");
+var seven = document.querySelector("#two-zero");
+var eight = document.querySelector("#two-one");
+var nine = document.querySelector("#two-two");
+
```

Comment:
Code for generating an array representation of a square tictactoe board
gameSize is the length of the board; "3" for a 3x3 board, "4" for a 4x4 board
gameBoard will be the generated array which you can use to represent your tictactoe
In this example the array will be filled with "0" in each cell
You could then write code that changes the value of a cell to smth like "1" or "2" to represent X and O.
```suggestion
var gameSize = 3;
var gameBoard;
for (let i = 0; i < gamesize; i++) {
    var gameRow;
    for (let i = 0; i < gamesize; i++) {
        gameRow.push(0);
    }
    gameBoard.push(gameRow);
}
```